### PR TITLE
docs(linux): update wiki

### PR DESCRIPTION
**Pull request by Meister1593**
General cleanup and updates,  and making sure users know where wiki for linux issues is.

### CODE REVIEW

File: wiki/Linux-Troubleshooting.md
Comment (The-personified-devil) [inline wiki/Linux-Troubleshooting.md:38]: it isn't exhaustive but it's like 3 distros or something and we should at least mention them
and potentially include fixes or provide a general explanation on what to look for to fix it

File: wiki/Linux-Troubleshooting.md
Comment (The-personified-devil) [inline wiki/Linux-Troubleshooting.md:15]: Does this not do the exact same thing as the other xwayland fix just through a different mechanism or am I missing something (ik you didn't change it I just never got around to address it)

File: wiki/Linux-Troubleshooting.md
Comment (The-personified-devil) [inline wiki/Linux-Troubleshooting.md:41]: would be nice to also include how to check for amdgpu-pro since we should only need to extend the grep

**Review by The-personified-devil — CHANGES_REQUESTED** [review]

File: wiki/Linux-Troubleshooting.md
Comment (Meister1593) [reply wiki/Linux-Troubleshooting.md:2]: Oops was a bit sleepy

File: wiki/Linux-Troubleshooting.md
Comment (Meister1593) [reply wiki/Linux-Troubleshooting.md:15]: Afaik its a bit different thing and specific to wlroots, not sure what exactly but other De's are not really affected

File: wiki/Linux-Troubleshooting.md
Comment (Meister1593) [reply wiki/Linux-Troubleshooting.md:41]: From what I can tell, its starting with same name (amd_icd), but not sure If its in the same place (might be in /opt, but idk how vulkan behaves in this case, how does it know where to look??)

File: wiki/Linux-Troubleshooting.md
Comment (The-personified-devil) [reply wiki/Linux-Troubleshooting.md:15]: then it's probably some mess due to steamvr being built on 50 different things at the same time

File: wiki/Linux-Troubleshooting.md
Comment (The-personified-devil) [reply wiki/Linux-Troubleshooting.md:41]: https://wiki.gentoo.org/wiki/AMDGPU-PRO
This lists them for gentoo, no clue for arch. But tbh I actually think we can ignore that in the detection because you must do some pretty cursed shit to install amdgpu-pro. And in that case the user can go figure it out themselves
But fwiw, I think that path always needs to be one or rather it's always one centralized path, even if it might vary between distros

Comment (The-personified-devil) [reply wiki/Linux-Troubleshooting.md:41]: arch too seems to install it as amd_pro `mv "${pkgdir}"/opt/amdgpu-pro/etc/vulkan/icd.d/amd_icd64.json "${pkgdir}"/usr/share/vulkan/icd.d/amd_pro_icd64.json`.
For ubuntu (that what this aur script is repacking it seems to be a much bigger mess however), but since arch is like 80% of linux gaming I think just checking for amd_pro too should be enough

File: wiki/Linux-Troubleshooting.md
Comment (zmerp) [reply wiki/Linux-Troubleshooting.md:2]: Let's also capitalize ALVR

**Review by zmerp — COMMENTED** [review]
No comments from me. This is mainly Linux documentation, so I'll wait for @The-personified-devil  approval

**Comment by Meister1593** [comment]
OK From me for @The-personified-devil changes

**Comment by The-personified-devil** [comment]
Seems we need someone to properly review this so I can merge this due to github moment

**Comment by Meister1593** [comment]
Ok this didn't work lol

**Comment by Meister1593** [comment]
@zmerp can you please approve it? seems like someone contributing to PR can't be approving person

**Review by zmerp — APPROVED** [review]

**Pull request merged**
